docs/source/contributing/getting-started.rst

```diff
@@ -0,0 +1,329 @@
+
+      cd substra
+      pip install -e .
+
+6. Install frontend dependencies
+
+   .. code-block:: bash
+
+      cd substra-frontend
+      npm install --dev
+
+Launching
+=========
+
+* Deploy the orchestrator
+
+  .. code-block:: bash
+
+   cd orchestrator
+   skaffold run
+
+.. _Deploy the backend:
+
+* Deploy the backend
+
+  .. code-block:: bash
+
+   cd substra-backend
+   skaffold run
+
+  .. caution::
+     On arm64 architecture (e.g. Apple silicon chips M1 & M2), you need to add the profiles ``dev`` and ``arm64``.
+
+     .. code-block:: bash
+
+      skaffold run -p dev,arm64
+
+.. tip::
+   When re-launching the orchestrator and the backend, you can speed up the processing by avoiding the update of the chart dependencies using the profile ``nodeps``.
+
+   .. code-block:: bash
+
+      skaffold run -p nodeps
+
+* Deploy the frontend. You can use two methods (described below)
+
+  a. Local server: Execute the following command:
+
+    .. code-block:: bash
+
+      API_URL=http://substra-backend.org-1.com npm run dev
+
+  b. Docker:
+
+     .. code-block:: bash
+
+      docker build -f docker/substra-frontend/Dockerfile --target dev -t substra-frontend .
+      docker run -it --rm -p 3000:3000 --name DOCKER_FRONTEND_CONTAINER_NAME -e API_URL=http://substra-backend.org-1.com -v ${PWD}/src:/workspace/src substra-frontend
+
+     | with ``DOCKER_FRONTEND_CONTAINER_NAME`` the name of the frontend container that will be used for the rest of the operations.
+
+  * In both case, you can access the frontend at http://substra-frontend.org-1.com:3000/.
+
+Monitoring
+==========
+
+You can use kubectl_ command to monitor the pods. Tools like `k9s <https://github.com/derailed/k9s>`_ and `k8lens <https://k8slens.dev/>`_ provide graphical interfaces to monitor the pods and get their logs.
+
+Stopping
+========
+
+To stop the Substra stack, you need to stop the 3 components (backend, orchestrator and frontend) individually.
+
+* Stop the frontend: This action depends on which option you chose during the launch:
+
+  a. Local server: Stop the process running the local server (usually using *Control+C* or *Command+C* on macOS)
+  b. Docker:
+
+     .. code-block:: bash
+
+        docker stop DOCKER_FRONTEND_CONTAINER_NAME
+
+     | with ``DOCKER_FRONTEND_CONTAINER_NAME`` the name of the frontend container you chose during the launch
+* Stop the orchestrator:
+
+  .. code-block:: bash
+
+     cd orchestrator
+     skaffold delete
+
+* Stop the backend:
+
+  .. code-block:: bash
+
+     cd substra-backend
+     skaffold delete
+
+If this command fails and you still have pods up, you can use the following command to remove the ``org-1`` and ``org-2`` namespaces entirely.
+
+.. code-block:: bash
+
+   kubectl rm ns org-1 org-2
+
+Next steps
+==========
+
+Now you are ready to go, you are ready to run either the :doc:`/auto_examples/index` or the :doc:`Substrafl (low-level library) examples </substrafl_doc/examples/index>` (low-level library).
```

Comment:
```suggestion
Now you are ready to go, you are ready to run either the :doc:`/auto_examples/index` or the :doc:`Substrafl examples </substrafl_doc/examples/index>`.
```

Reply:
If you point to the substrafl examples, we need to add the lib above to clone the github and to install it